visma/matrix/operations.py

```diff
@@ -220,3 +220,21 @@ def scalarDiv(const, mat):
         return matRes
     else:
         logger.error("ZeroDivisionError: Cannot divide matrix by zero")
+
+
+def transposeMat(mat):
```

Comment:
Can you define this as a class method of _Matrix_ class in **matrix/structure.py** instead of defining it in **operations.py**?

Reply:
Sure. I thought since of placing it here since we were placing all matrix operations in here.
But I'd do that.

Reply:
@8hantanu my bad didn't see there is already a defined transpose method inside Matrix class.

Although, I don;t quite understand the purpose of RHS argument.

Reply:
I too don't remember why I had put it there :sweat_smile:  Just use the single `self `argument.